partcad/tests/unit/test_assembly.py

```
@@ -31,31 +31,31 @@ def test_assembly_primitive():
 
 def test_assembly_example_assy_primitive():
     ctx = pc.init("examples")
-    primitive = ctx._get_assembly("/produce_assembly_assy:primitive")
+    primitive = ctx._get_assembly("//produce_assembly_assy:primitive")
     assert primitive is not None
     assert asyncio.run(primitive.get_wrapped(ctx)) is not None
     assert asyncio.run(primitive.get_build123d(ctx)) is not None
 
 
 def test_assembly_example_assy_logo():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_assembly_example_assy_logo`

Add return type annotation: `None`

(ANN201)